host/src/pages/GameInProgress.jsx

```diff
@@ -62,13 +66,23 @@ export default function GameInProgress({
   const correctChoiceIndex =
     questionChoices.findIndex(({ isAnswer }) => isAnswer) + 1;
   const statePosition = Object.keys(GameSessionState).indexOf(currentState);
+
   // using useMemo due to the nested maps in the getAnswerByQuestion and the fact that this component rerenders every second from the timer
   const answers = useMemo(
```

Comment:
here's that place in GameInProgress where the new answer object split takes place